FEM/fem_ele_vec.cpp

```diff
@@ -1149,9 +1116,13 @@ void CFiniteElementVec::LocalAssembly(const int update)
 			if (static_cast<size_t>(pcs->ExcavMaterialGroup) == MeshElement->GetPatchIndex())
 			{
 				double const* ele_center(MeshElement->GetGravityCenter());
-				if ((GetCurveValue(pcs->ExcavCurve, 0, aktuelle_zeit, &valid) + pcs->ExcavBeginCoordinate)
-				        > (ele_center[pcs->ExcavDirection])
-				    && (ele_center[pcs->ExcavDirection] - pcs->ExcavBeginCoordinate) > -0.001)
+				const double expected_coordinate =
```

Comment:
isn't it possible to use the new function `isPointInExcavatedDomain()` here?

Reply:
Replace the lines with the new function.